.travis/package.sh

```diff
@@ -7,6 +7,5 @@ tar -czf "$filename" \
     -C export/ \
     css \
     fonts \
-    images \
```

Comment:
Must have silently been failing anyway because of this.

Reply:
That said, easier to just tar the whole directory?

Probably also need to include the LICENSE file.

Reply:
(One for later anyway.)